data/aliases.ts

```diff
@@ -2509,35 +2605,63 @@ export const CompoundWordNames = [
 	'Zap-dos',
 
 	// moves
+	'10,000,000 Volt Thunder-bolt',
 	'Accele-rock',
 	'Acid Down-pour',
+	'Acro-batics',
 	'Acu-pressure',
+	'Aero-blast',
+	'Allur-ing Voice',
```

Comment:
This one definitely doesn't make sense. I don't think anyone says "aiv" or "aing" or "ing".